knowledge-base/adr/0017-code-snippets.ts

```diff
@@ -0,0 +1,127 @@
+/* eslint-disable */
+namespace Classes{
```

Comment:
Not sure if this really needs to be here?

Reply:
I would like to keep it as an explanation what has been tried and explain the options.